module-2/backend/week_one.md

```diff
@@ -5,30 +5,127 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  Get  - pulls information from database to read
+  Post - creates a record in database
+  Put - updates a record in database - full replace
+  Patch - updates specific attributes 
+  Delete - destroys recrod 
+  
 2. What is Sinatra?
+  
+  Mildly opinionated framework for Ruby.
+ 
+
 4. What is MVC?
+
+  Model 
+  View
+  Controller 
+
+  Maintins single responsibility in our app. 
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+  So other people understand out code. 
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  
+   
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+  @count = 1
     erb :index
   end
   ```
+  
+  tasks = Task.all
+  erb :'tasks/index', :locals =>tasks:tasks
+  
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  
+  get '/horses' do
+  name = 'Mr. Ed'
+  erb :index, :locals =>{name: name}
+
 9. What's the purpose of ERB?
+
 10. Why do I need a development AND test database?
+  
+ The test database is going to be continulally torn down after our tests are run, whereas the development database gives us something to work with in Tux or Pry etc. etc. 
+
 11. What's responsive design?
+
+ Responsive design refers to design that takes into account the fact that differernt users will be interacting with your app on different devices with defferent screen sizes. 
+
 12. What is CRUD and why is it important?
+
+CRUD is the data life cycle - Creat, Read, Update, Delete; CRUDing out a resource requires the developer to think through how data is coming in, and how the user is interacting with it, which is important because it prevents the developer from neglecting any of the lifecycle pieces. 
+  
 13. What does HTTP stand for? 
+
+  Hyper(T)ext Transfer Protocol 
+
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+  
+  <%= class.attributename %>  
+  "#{some text}"
+

```

Comment:
the two ways are the first way that you listed and <% ruby %>. The first allows the information to actually display on the page while the other allows us the access to our ruby without explicitly displaying it.